packages/node/src/integrations/tracing/fastify.ts

```diff
@@ -131,3 +144,10 @@ function addFastifySpanAttributes(span: Span): void {
     span.updateName(name.replace(/^fastify -> /, ''));
   }
 }
+
+/** Returns true if response code is internal server error */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function defaultShouldHandleError(_request: any, reply: any, _error: Error): boolean {
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+  return reply.statusCode >= 500;
```

Comment:
statusCode can be undefined

Reply:
// eslint-disable-next-line is used everywhere, it is not clear why you need eslint in the project then.

Reply:
> statusCode can be undefined

Could probably implement this similar to what's been done in the [express integration](https://github.com/getsentry/sentry-javascript/blob/6e4ee137717bd3b8ca37469d73a54e9dfb5218a3/packages/node/src/integrations/tracing/express.ts#L148-L157).